rest/src/main/webapp/js/entry.js

```diff
@@ -68,6 +68,16 @@
                             value.isEdited = true;
                         };
 
+                        $scope.toggleHistoryViewer = function (value)
+                        {
+                            $window.getSelection().removeAllRanges();
+                            if (value.isHistory == null) {
```

Comment:
Can we call this something like 'shouldDisplayHistory' or something?

Reply:
Will do, just a sec